### ISSUE TITLE
show faf version at webfaf footer

### ISSUE THREAD

**Issue by mjtrangoni** [abrt/faf#664]
Hi @mhabrnal, @marusak,

I think, it would be nice showing the FAF version at the webfaf footer. Instead of `FAF | About | Report a bug`, it could be: 
 1. `FAF v1.3.1 | About | Report a bug `
 1. Or even better, `FAF v1.3.1 (commit $(git rev-parse --short refs/tags/1.3.1) | About | Report a bug `

**Comment by marusak** [comment]
Good idea. Thanks for it. We will implement when there will be some spare time. Or do you want to take a look at it? :)

**Comment by marusak** [comment]
Sure, when we will have the version somewhere accessible, then it can be also integrated in `faf` command line tool.

**Comment by Sylvia23** [comment]
Hey! Can I take up this issue to work upon if its free? This might help me to get started and set up the development environment. Also , any advice on where and how to get started with the implementation is also most welcome! Thanks!

**Comment by marusak** [comment]
Hi @Sylvia23 
I believe this issue is free, unless @mjtrangoni works on it.
As has been mentioned it would also make sense to implement `faf --version` command so lets assume both of these options. Both variants however need to know the current version. To be honest I am not entirely sure where we should get this information from. The first idea that come to my mind is to have it in configuration file. So if you don't have a better idea you may start with that. Let me give you some hints:
1) Create a new variable in `config/faf.conf.in` (for example `Version = '1.2.3'`, this will need to be set up automatically while building, but for now it is fine)
2) Implementing `faf --version` - look into `src/bin/faf` and see how you can check if argument `--version` was passed and then only print version that is read from the config file
3) Implement showing version in footer - look into `src/webfaf/templates/base.html`

**Comment by mjtrangoni** [comment]
@Sylvia23 you can work on it. @marusak I agree with you.

**Comment by sanjana098** [comment]
Hi, is this issue still to be fixed? I'm interested to work on this. Please let me know if that's possible. 
Thanks in advance!

**Comment by marusak** [comment]
Hi @sanjana098 
I think @Sylvia23 has abandon this issue, so feel free to work on it.

**Comment by anshultale** [comment]
Hi @marusak, I would like to work on this if it's not already picked up.

**Comment by sanjana098** [comment]
Hi @anshultale, I am working on this right now. 😅

**Comment by sanjana098** [comment]
Hi, this issue is free to be taken. Sorry for the delay!

**Comment by anshultale** [comment]
Hi @sanjana098  , I will work on this issue.

Thanks

**Comment by LucianaMarques** [comment]
Has any PR been made on this? I'd like to contribute.
